### PR TITLE
Added few more flags with information about pod.

### PR DESCRIPTION
**Pull request by loburm**
Add flag pod-id and namespace-id that are used to create correct
monitored resource object for the stackdriver.

Additionally some small refactoring to improve readability.

### CODE REVIEW

File: prometheus-to-sd/config/pod_config.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+// ContainerConfig describes pod in which current component is running.
+type ContainerConfig struct {
```
Comment (piosz) [inline prometheus-to-sd/config/pod_config.go]: Shouldn't be `PodConfig`? In the comment you have "describes pod".

**Review by piosz — COMMENTED** [review]
LGTM, just a nit.

Could you please send refactoring as a separate commit/pr in the future?

**Comment by loburm** [comment]
Sorry that was unplanned refactoring. Next time I'll do it in other commit.

I have renamed ContainerConfig to PodConfig. Thanks for the comment.

**Comment by piosz** [comment]
@mwielgus could you please approve flags?

**Pull request merged**